src/bootstrap/doc.rs

```diff
@@ -374,9 +374,15 @@ impl Step for Standalone {
             }
 
             if filename == "not_found.md" {
-                cmd.arg("--markdown-css").arg("https://doc.rust-lang.org/rust.css");
+                cmd.arg("--markdown-css")
+                    .arg(format!("https://doc.rust-lang.org/rustdoc{}.css", &builder.version))
+                    .arg("--markdown-css")
+                    .arg("https://doc.rust-lang.org/rust.css");
```

Comment:
This doesn't seem right, that file will go away as soon as this PR is merged. Why was it necessary? That makes me think the whole approach isn't feasible.

Reply:
Oh wait I misunderstood what's going on - you're removing *most* of rust.css and only leaving the things specific to the website. Ignore me.